Bluetooth communication signoff

CarsonDPope opened the pull request.

CarsonDPope committed:
Create readme.md

CarsonDPope committed:
Create readme.md

CarsonDPope committed:
Create readme.md

CarsonDPope committed:
Create Bluetooth_Communication.md

CarsonDPope committed:
Update Bluetooth_Communication.md

CarsonDPope committed:
Update Bluetooth_Communication.md

CarsonDPope committed:
Update Bluetooth_Communication.md

CarsonDPope committed:
Update Bluetooth_Communication.md

CarsonDPope committed:
Update Bluetooth_Communication.md

CarsonDPope committed:
Add files via upload

CarsonDPope committed:
Update Bluetooth_Communication.md

CarsonDPope committed:
Update Bluetooth_Communication.md

CarsonDPope committed:
Update Bluetooth_Communication.md

CarsonDPope committed:
Add files via upload

CarsonDPope committed:
Delete Documentation/Images/Main-Processor/Buildable-Schematics/DSPBOARD_REV3.png

CarsonDPope committed:
Update Main_Processor.md

CarsonDPope committed:
Update Bluetooth_Communication.md

CarsonDPope committed:
Add files via upload

CarsonDPope committed:
Add files via upload

CarsonDPope committed:
Delete Documentation/Electrical/Schematics/DSPModel_REV3_fixedViewport.pdf

CarsonDPope committed:
Add files via upload

CarsonDPope committed:
Delete Documentation/Electrical/Schematics/Sources/DSPModel_REV3.dwg

CarsonDPope committed:
Update Bluetooth_Communication.md

JesseTNRoberts reviewed (commented):
No time delay constraints seems a problem.

CarsonDPope committed:
Update Bluetooth_Communication.md

JesseTNRoberts reviewed (commented):
I am a bit confused why this does not also have to abide by the 1.4 ms time budget. I think I know, and I think it is because you are recalculating filter parameters but executing the filter locally.

At any rate, be sure to make the situation clear.

CarsonDPope commented:
You are exactly correct, I will work on making that more clear

CarsonDPope committed:
Update Bluetooth_Communication.md

CarsonDPope committed:
Update Bluetooth_Communication.md

CarsonDPope committed:
Update Lessons Learned.md

CarsonDPope committed:
Update Bluetooth_Communication.md

JesseTNRoberts commented on Documentation/Signoffs/Bluetooth_Communication.md:
```diff
@@ -0,0 +1,95 @@
+#### Bluetooth Capabilities
+The Arduino UNO R4 WiFi is able to transmit and receive in WiFi and Bluetooth 5 LE<sup>2</sup>. This is accomplished through the ESP32-S3-MINI[3]. Bluetooth 5.0 LE includes a PHY called LE 2M. This can support 2 Mb/s (2,000,000 bits per second). At the application layer, a data rate of up to approximately 1.4 Mb/s (1,400,000 bits per second) is possible[5]. Assuming the max 2 second delay lines o ...
+
+spi here
```
I think something is missing?

JesseTNRoberts commented on Documentation/Signoffs/Bluetooth_Communication.md:
```diff
@@ -0,0 +1,95 @@
+
+spi here
+
+Since the subsystem is able to communicate wirelessly, the physical extent of the system resides at the window preventing any obstruction of pathways<sup>9</sup>. The room's dimensions are 34x24x11 feet. A worst case scenario (lower corner to opposite upper corner) is a distance of 43.05 ft (13.122 meters). The ESP32-S3-MINI has a typical Bluetooth transmit power of ** dBm and a typical receiver  ...
```
**?

CarsonDPope committed:
Update Bluetooth_Communication.md

JesseTNRoberts commented on Documentation/Signoffs/Bluetooth_Communication.md:
```diff
@@ -0,0 +1,93 @@
+
+## Analysis
+
+#### Power
```
I'm not sure that this shows the constraint is met?

JesseTNRoberts commented on Documentation/Signoffs/Bluetooth_Communication.md:
```diff
@@ -0,0 +1,93 @@
+#### Bluetooth Capabilities
+The Arduino UNO R4 WiFi is able to transmit and receive in WiFi and Bluetooth 5 LE<sup>2</sup>. This is accomplished through the ESP32-S3-MINI[3]. Bluetooth 5.0 LE includes a PHY called LE 2M. This can support 2 Mb/s (2,000,000 bits per second). At the application layer, a data rate of up to approximately 1.4 Mb/s (1,400,000 bits per second) is possible[5]. Assuming the 2 second delay lines of 18 ...
+
+Since the subsystem is able to communicate wirelessly, the physical extent of the system resides at the window preventing any obstruction of pathways<sup>9</sup>. The room's dimensions are 34x24x11 feet. A worst case scenario (lower corner to opposite upper corner) is a distance of 43.05 ft (13.122 meters). The ESP32-S3-MINI has a typical Bluetooth transmit power of 0 dBm (but can be set higher)  ...
```
This is missing a heading

JesseTNRoberts commented on Documentation/Signoffs/Bluetooth_Communication.md:
```diff
@@ -0,0 +1,93 @@
+The system can be powered through a USB connection or through the onboard barrel jack connector[1] <sup>1</sup>. The recommended input voltage and current rating for the external power supply are 6-24 V. The subsystem will use the included AC/DC adapter as its power supply. Since the board needs to use the USB port in order to install the designed software, the subsystem will have to be programme ...
+
+#### Bluetooth Capabilities
+The Arduino UNO R4 WiFi is able to transmit and receive in WiFi and Bluetooth 5 LE<sup>2</sup>. This is accomplished through the ESP32-S3-MINI[3]. Bluetooth 5.0 LE includes a PHY called LE 2M. This can support 2 Mb/s (2,000,000 bits per second). At the application layer, a data rate of up to approximately 1.4 Mb/s (1,400,000 bits per second) is possible[5]. Assuming the 2 second delay lines of 18 ...
```
This is greater than the constraint. Should the constraint be changed? If so, explain why in the constraint section and make the change.

CarsonDPope committed:
Update Bluetooth_Communication.md

This go around, I am not done with SPI speed and dont have time to change the time constraint in writing. New constraint is made just needs to be typed out.

Also, schematic needs to be changed as well.

JesseTNRoberts reviewed (commented):
Same comments

CarsonDPope committed:
Add files via upload

CarsonDPope committed:
Add files via upload

CarsonDPope committed:
Add files via upload

CarsonDPope committed:
Update Bluetooth_Communication.md

CarsonDPope committed:
Add files via upload

CarsonDPope committed:
Add files via upload

CarsonDPope committed:
Add files via upload

CarsonDPope committed:
Update Main_Processor.md

CarsonDPope committed:
Update Bluetooth_Communication.md

JesseTNRoberts reviewed (approved):
Looks good.
